core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -345,7 +372,10 @@ public HttpRequestInitializer create() {
       }
       if (credentials.createScopedRequired()) {
         credentials =
-            credentials.createScoped(Collections.singletonList(SQLAdminScopes.SQLSERVICE_ADMIN));
+            credentials.createScoped(Arrays.asList(
```

Comment:
Any reason to switch from `Collections` to `Arrays`?

Reply:
Nope. it was mostly to move away from the singleton list to add another scope. If we're using the separate tokensource factory I think this is actually unneccessary, but when we eventually get the getter added to `HttpCredentialsAdapter`, we'll want the credentials object to have both scopes